galeracluster/source/training/tutorials/getting-started-docker.rst

```diff
@@ -77,7 +77,7 @@ In Docker, Dockerfiles are used to describe the Docker images we are going to us
    RUN apt-key adv --recv-keys --keyserver hkp://keyserver.ubuntu.com:80 8DA84635
    RUN add-apt-repository 'deb https://releases.galeracluster.com/ubuntu trusty main'
    RUN apt-get update
-   RUN apt-get install -y galera-3 galera-arbitrator-3 mysql-wsrep-5.6 rsync lsof
+   RUN apt-get install -y galera-3 galera-arbitrator-3 mysql-wsrep-8.0 rsync lsof
```

Comment:
i do not think the file is galera-arbitrator-3

Reply:
Changed to galera-arbitrator-4